zap/src/main/java/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -347,9 +351,9 @@ private void setupLookAndFeel() {
             return;
         }
 
-        if (Constant.isMacOsX()) {
-            OsXGui.setup();
-        } else if (setLookAndFeel(getLookAndFeelClassname("Nimbus"))) {
+        // Default to "Nimbus" LookAndFeel class for non non macOS systems. Nimbus does not fit well
```

Comment:
Just one “non” :wink:

Reply:
Is the comment really needed? The later part is not accurate, it's not that does not fit well there's one that fits better.

Reply:
Okay I've took out the comment